src/jobs/npm_security_audit.yml

```diff
@@ -1,7 +1,9 @@
 ---
 description: >
   Node.js projects using npm - auditing package dependencies for security vulnerabilities
-executor: node
+executor:
+  name: node
+  tag: "16.15"
```

Comment:
[Previously](https://github.com/ministryofjustice/hmpps-circleci-orb/pull/116/files) we used the `-browsers` version, we're switching to the regular one because we don't need browser drivers?